### ISSUE TITLE
Training Scheme on Waymo

### ISSUE THREAD

**Issue by Brummi** [YihongSun/Dynamo-Depth#11]
Dear authors,

thanks for sharing your nice work!
I am currently also trying to train Monodepth2 on waymo and I am running into some stability issues.

Did you modify the training code for Monodepth2 in any way when training it on Waymo?

Many thanks in advance!
Best,
Felix

**Comment by YihongSun** [comment]
Hi Felix, 

Thank you for your interest in our work!

As far as I can recall, when evaluating Monodepth2 on Waymo Open, we just followed the official codebase and training procedure. From our investigations, small variations in learning rate, schedule, and number of epochs didn't appear to affect Monodepth2 performance at convergence.

Can you elaborate on the stability issue that you are running into? From our experience, the predominate errors that we observed are for dynamical regions, as shown by Figure 2 in the [paper](https://arxiv.org/pdf/2310.18887).

Hope this helps!

**Comment by YihongSun** [comment]
Closing this issue due to inactivity. Feel free to reach out if you have additional questions!
